src/region/RegionAnnotations.tsx

```diff
@@ -73,6 +70,15 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         }
 
         createRegion({ ...staged, message });
+        this.resetStaged();
```

Comment:
Would we want to do this only after the API returns successfully?

Reply:
Yeah, good point, we shouldn't reset the status here.